deployment/terraform/cdn.tf

```diff
@@ -38,6 +38,12 @@ resource "aws_cloudfront_distribution" "cdn" {
     min_ttl                = 0
     default_ttl            = 0
     max_ttl                = 300
+
+    ## This feature is only available in Terraform AWS Provider 3.41.0 and later
+    # function_association {
+    #   event_type   = "viewer-request"
+    #   function_arn = "${aws_cloudfront_function.oshub_conditional_redirect.arn}"
+    # }
```

Comment:
Left behind for future reference on how this could be automatically provisioned if we have a newer version of Terraform and the AWS provider.